_posts/2022-03-16-hhvm-4.153.markdown

```diff
@@ -0,0 +1,18 @@
+---
+title: "HHVM 4.153"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.153 is released! This release marks the end of support of 4.146; 4.147 -
+4.151 remain supported, as do the 4.102 and 4.128 LTS releases. HHVM 4.152 was
+skipped due to issues with the nightly builds.
```

Comment:
This release should be an LTS, and 4.102 becomes unsupported.

HHVM 4.152 was also scheduled to be an LTS release. This release should be an LTS release, and is expected to be supported for 47 weeks - usual would be 48, but as it's a week late, we're cutting off - even though we move stuff around a little bit, we keep the target dates constant.

The LTS announcement also seems like a good time to switch to a 2-week regular release schedule; should be clear that this is an experiment, and we will be looking for feedback. Reverting to weekly or moving to monthly are on the table - we want to understand how this affects our users